### DISCUSSION TITLE
Using iSpindel's Temperature for the Beer Temp

### DISCUSSION THREAD

**Discussion by MonTage69** [thorrak/fermentrack#628] [Q&A] [answered]
I have a Fermzilla All rounder, and pressure ferment. Currently i am only using Chamber (fridge) Temp as the controlling factor of the heating and cooling. I know some people tape a temp probe to the side of the vessel and insulate it from the external temps. However i find it would be much better to use the iSpindels Temp, as its submerged in the Wort/Beer. There is no way currently to do this

Not sure if this would be a BrewPI (probably more a brewpi) Question or Fermentrack question, but i thought i would ask on both.

**Comment by thorrak** [comment, marked as answer]
This gets asked periodically, and unfortunately, this is something that I don't plan to build out into Fermentrack/BrewPi-ESP for the iSpindel. WiFi is a battery hog which is why the iSpindel only checks in every few minutes. The algorithm built into the BrewPi firmware expects significantly more frequent temperature readings than this, and as such I don't expect that the user experience would ultimately be a positive one.

That said, with the advent of the Tilt Pro, the Tilt now provides tenth-of-a-degree temperature readings every ~1.2 seconds. Assuming I can get it to work reliably over Bluetooth, I plan to build out support for doing this with a Tilt Pro.

**Reply by MonTage69** [reply]
That does make a lot of sense. This gets my thoughts going.

**Comment by uSlackr** [comment]
I wanted to do the same thing when I first got my tilt and found Fementrack.  Then I realized FT is made for tight temp control using PID methods.  Once I understood that, I built a brewpi device and use a thermowell to get temps.


\\Greg

**Reply by MonTage69** [reply]
I see now after some research that you can get a thermowell for the all rounder and apparently it still allows for pressure fermentation. 
Think this would be the easiest route. Thanks
